src/main/java/com/scrive/pdftools/tools/flattener/FlattenerHandler.java

```diff
@@ -0,0 +1,17 @@
+package com.scrive.pdftools.tools.flattener;
+
+import org.json.JSONObject;
+
+import java.io.ByteArrayOutputStream;
+
+public class FlattenerHandler {
+
+    public static ByteArrayOutputStream execute(JSONObject flatSpecJSON) throws Exception {
```

Comment:
the former question applies also here: do we need the BAOS or would returning a `byte[]` be sufficient?

Reply:
Nope. I changed it to return a byte array instead